### ISSUE TITLE
Need instructions for contributing to WCAG-ACT repository

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#17]
Provide contributing document and/or updated wiki page with instructions on how to contribute. Should submit pull requests to master branch.

**Comment by WilcoFiers** [comment]
@a11ydoer suggestion: adding this to the README.md document.

**Comment by moekraft** [comment]
I was thinking of a Contributing.md doc. I'll put together a draft.

**Comment by moekraft** [comment]
@WilcoFiers Discussed with Wilco and Shadi yesterday. Propose a Contributing.md. For example, see https://github.com/moekraft/Va11yS/blob/master/CONTRIBUTING.md. The contributing file should also link to [W3C Code of Ethics and Professional Conduct.](https://www.w3.org/Consortium/cepc/)

**Comment by WilcoFiers** [comment]
I've created a PR for this:
https://github.com/w3c/wcag-act/pull/30

**Comment by moekraft** [comment]
We have merged Wilco's pull request. I will generate a new request to generate suggested updates from review.

**Comment by moekraft** [comment]
Comments from PR #30 


1. I think the first heading level 2 should just be Provide Feedback.
2. We should add a link to ACT Task Force in the first sentence.
3. For better Accessibility, we should use link text for the following links:

https://github.com/w3c/wcag-act/issues should have link text "w3c/wcag-act Issues tab".
https://www.w3.org/WAI/GL/task-forces/conformance-testing/#contribute should have link text "How to Comment, Contribute and Participate"

4. Under Code of Conduct should be maintaining in place of maintain, "We are committed to maintaining a positive work environment."

**Comment by moekraft** [comment]
Comments on PR #30 from MJ. 
A few comments:
Leave Changes and Suggestions:

- is there a preferred way to leave comments on commits? Do we want comments left on the issue, or by using "Review Changes" on the file that was changed? We discovered last week that comments left on the files changed didn't propagate up to the issue that the change was being made for.

- Should  add a bullet for contributors to mark their content with the label "Ready to survey" by using the labels functionality in the right column. Should the label be added both to pull requests and issues?
How do we want editorial changes suggested? Is it better done in a pull request rather than describing minute details like that in a comment or issue?

Submit a pull request section: First sentence says "request from to pull changes" and I think it should say "request to pull changes".
Using GitHub - Do you have to be added to the W3 group to be able to contribute? Do you have to request to be in the wcag-act group to be able to contribute? I think you need to be added to be able to be assigned issues.

**Comment by moekraft** [comment]
Pull request #55 has been merged per Wilco's approval: Re: Updates to Contributing.md
Yesterday, 7:28 AM
Wilco Fiers to me, Accessibility Conformance Testing
Show more
Approved. Feel free to merge Moe.
